nomenclature/definitions/variable/emissions/emissions.yaml

```
@@ -290,6 +290,42 @@ Emissions|CO2|AFOLU:
                 (IPCC category 3)
    unit: Mt CO2/yr
 
+Emissions|CO2|Electricity:
+   description: CO2 emmissions of power plants .
```

typo, please correct throughout this document and remove `.`
```suggestion
   description: CO2 emissions of power plants
```